Remotion/Data/DomainObjects/Mapping/TypeDefinition.cs

```diff
@@ -0,0 +1,224 @@
+﻿#nullable enable
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Threading;
+using Remotion.Data.DomainObjects.Persistence.Model;
+using Remotion.Reflection;
+using Remotion.Utilities;
+
+namespace Remotion.Data.DomainObjects.Mapping
+{
+  public abstract class TypeDefinition
```

Comment:
Please add a testfixture for TypeDefinition with at leaast one empty method and an ignore with the appropriate todo-comment. That way, the missing tests popup.